app/Services/SpreadsheetImportService.php

```
@@ -145,8 +152,47 @@ private function resolveOpening(array $row, int $projectId, int $userId): Openin
+     * "CPF FISCAL" / "MATRICULA DO FISCAL" e um e-mail sintético,
+     * já que a planilha não traz e-mail do fiscal.
+     */
+    private function resolveSupervisor(array $row): ?User
+    {
+        $name = trim((string) ($row['FISCAL'] ?? ''));
+
+        if ($name === '') {
+            return null;
+        }
+
+        $user = User::where('name', $name)->first();
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\User::where().


(staticMethod.notFound)